script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_1.txt

```diff
@@ -30,40 +30,40 @@ there is nothing to be gained by seeing such things-----
 <Page8>
 "------hmnnn."
 I wake up from my old memories.
-I feel the morning light through my eyelids and crawl my hands under my pillow with my eyes closed.
+I feel the morning light through my eyelids and feel around on beside my pillow with my eyes closed.
```

Comment:
Spurious `on`

Reply:
Also, I feel like this changes the meaning a bit? He initially sticks his hands under to find his glasses, then to the right on the next page? Or was the initial tl here already taking liberties? I also don't love duplicating words too  close, so beside here + beside two lines down feels like a reword candidate

Reply:
IMO the line was originally mis-TL'd. Here's the JP:
> 　目蓋ごしに朝の光を感じ取って、目を閉じたまま枕元に手を這わせる。

枕元 doesn't even have the connotation of "under the pillow", my gut reaction is even "bedside" instead of "beside my pillow", but the next couple lines clarify it to be "the left of my pillow" specifically. Also regarding the duplicate word, it's after a page break so I think it's ok.

Reply:
I guess crawl/creep makes me think under, but yeah not explicit.